raspberry/bzzz/sensors/gnss.py

```diff
@@ -187,34 +344,73 @@ def latitude(self):
                                                  cursor=self.__cursor)
 
     @property
-    def longitude(self):
+    def relative_east(self):
         """
         Returns Longitude position in decimal
         """
         with self.__lock:
             return self.__data_processor.process(self.__values_cache[:, 1], 
                                                  cursor=self.__cursor)
+        
+    @property
+    def relative_down(self):
+        """
+        Returns Longitude position in decimal
```

Comment:
in decimal? you mean in metres?

Reply:
metres I forgot to change that description